config/environments/development.rb

```diff
@@ -36,6 +36,8 @@
   # Store uploaded files on the local file system (see config/storage.yml for options).
   config.active_storage.service = :local
 
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+
```

Comment:
We might also need to add letter_opener on development or add smtp configuration for sending mails

Reply:
sure